packages/auth/src/graphql/schema.graphql

```diff
@@ -119,6 +121,17 @@ type Access implements Model {
   createdAt: String!
 }
 
+type Subject implements Model {
+  "Unique identifier of the subject object."
+  id: ID!
+  "Wallet address of the subject's account."
+  subId: String!
+  "Format of the subject identifier, which is 'url'."
+  subIdFormat: String!
```

Comment:
```suggestion
  "Format of the subject identifier
  subIdFormat: String!
```
Or we specify that it should be `uri`. I lean towards not specify in the schema (if its not an Enum) so the docs don't become out of date with the actual application logic